src/isIsogram.test.js

```diff
@@ -6,4 +6,39 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`'word' has 1 repeating letter`, () => {
```

Comment:
The summary is not clear. Rewrite it because it would be hard to maintain this test in the future.